src/utils/hibp.js

```diff
@@ -18,49 +17,49 @@ function _addStandardOptions (options = {}) {
   const hibpOptions = {
     headers: {
       'User-Agent': HIBP_USER_AGENT
-    },
-    responseType: 'json'
+    }
   }
   return Object.assign(options, hibpOptions)
 }
 
-async function _throttledGot (url, reqOptions, tryCount = 1) {
-  let response
+async function _throttledFetch (url, reqOptions, tryCount = 1) {
```

Comment:
@mansaj note this may improve/reduce the amount of triggered errors (handling certain http status codes before throwing).  However, I have not refactored this.  As discussed, this should be refactored to throttle more elegantly.